src/kOS/Binding/FlightControlManager.cs

```diff
@@ -319,16 +344,28 @@ public bool Enabled
                 {
                     SafeHouse.Logger.Log(string.Format("FlightCtrlParam: Enabled: {0} {1} => {2}", name, enabled, value));
 
-                    enabled = value;
-                    if (steeringManager != null)
+                    if (enabled != value)
                     {
-                        if (enabled) steeringManager.EnableControl(shared);
-                        else steeringManager.DisableControl();
-                        //steeringManager.Enabled = enabled;
-                    }
-                    if (RemoteTechHook.IsAvailable(control.Vessel.id))
-                    {
-                        HandleRemoteTechPilot();
+
+                        enabled = value;
+                        if (name == "steering")
+                        {
+                            SafeHouse.Logger.Log("FlightCtrlParam: toggle steering parameter");
+                            IFlightControlParameter param = kOSVesselModule.GetInstance(shared.Vessel).GetFlightControlParameter("steering");
+                            if (enabled)
+                            {
+                                param.EnableControl(shared);
+                            }
+                            else
+                            {
+                                param.DisableControl(shared);
+                            }
+                            return;
+                        }
+                        if (RemoteTechHook.IsAvailable(control.Vessel.id))
```

Comment:
That's one of the things on the docket as part of my 3rd party api implementation.  One of the things I'd like to do is essentially create a way for dll's to hook our factory methods.  The other long goal here is that `FlightControlManager` stops handling the events itself, and that we move all of that function to `kOSVesselModule`.